Week-5/Work & HW/index/blackjack.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Black Jack</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <style>
+    #alert {
+      font-size: 300px;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="top">
+    </div>
+    <h1>Your Hand</h1>
+    <div id="div">
+    </div>
+    <button id="hit">Hit</button>
+    <button id="stay">Stand</button>
+    <div id="div2">
+      <h1>Dealer</h1>
+    </div>
+
+    <script>
+      let cards = [];
+      let cardsimages = [];
+      let cards2 = {};
+      cards2.cards = [];
+      cards2.total;
+      let cardsTotal;
+      let cards2images = [];
+      let checkNum = (num, array) => {
```

Comment:
THis is a weird method name: `checkNum`? It really should be called `addCardToListOfCards`